lib/index.js

```diff
@@ -1,17 +1,18 @@
 const fs = require('fs');
-const path = require('path');
+const cosmiconfig = require('cosmiconfig');
 const {
   fetchLabels,
   createLabels,
   deleteLabels,
   updateLabels,
 } = require('./api');
 
+const explorer = cosmiconfig('label', { searchPlaces: ['label-config.json'] });
```

Comment:
```suggestion
const explorer = cosmiconfig('github-labels');
```

This would mean that the config will look for

```js
[
  `package.json`,
  `.github-labelsrc`,
  `.github-labelsrc.json`,
  `.github-labelsrc.yaml`,
  `.github-labelsrc.yml`,
  `.github-labelsrc.js`,
  `github-labels.config.js`,
]
```

https://www.npmjs.com/package/cosmiconfig#searchplaces

Reply:
And we need to rename our config file of course 😉

Reply:
👍 
f17bfc0